webapp/channels/src/components/drafts/draft_row.tsx

```diff
@@ -266,6 +280,23 @@ function DraftRow({
         actions = draftActions;
     }
 
+    let title: React.ReactNode;
+    if (channel) {
+        title = (
+            <DraftTitle
+                type={(rootId ? 'thread' : 'channel')}
+                channel={channel}
+                userId={user.id}
+            />
+        );
+    } else {
+        title = (
+            <PlaceholderScheduledPostsTitle
```

Comment:
And continuing that same idea here, could we combine the behaviour of `PlaceholderScheduledPostsTitle` into `DraftTitle`? It already changes the text based on the type of the channel, so I think it could have a case for if the channel doesn't exist

Reply:
I wanted to do the same but making channel ID optional in DraftTitle required a lot of refactoring, and because we never show drafts for channels you don't belong to, all that refactoring would be a waste of time, hence, I didn't do it. Current way is the simplest.

Reply:
It might've been simple at the moment, but it does mean that the logic for "what is the title for the draft" is now spread across 2-3 files depending on if you count the `mapStateToProps` for the `DraftTitle` in there. That said, I worry I'm getting a bit too perfectionist in this case, so I'm fine leaving this as-is

Reply:
Preferring to keep it as-in